files/setup.sh

```diff
@@ -37,7 +37,6 @@ WEBHOOK_USERNAME=$(/root/setup/getOvfProperty.py "guestinfo.webhook_username")
 WEBHOOK_PASSWORD=$(/root/setup/getOvfProperty.py "guestinfo.webhook_password")
 CUSTOM_VEBA_TLS_PRIVATE_KEY=$(/root/setup/getOvfProperty.py "guestinfo.custom_tls_private_key")
 CUSTOM_VEBA_TLS_CA_CERT=$(/root/setup/getOvfProperty.py "guestinfo.custom_tls_ca_cert")
-DOCKER_NETWORK_CIDR=$(/root/setup/getOvfProperty.py "guestinfo.docker_network_cidr")
```

Comment:
are we breaking users with not having this option anymore?

Reply:
This PR overall is a breaking change :) 

This Network CIDR here was only applicable because Docker had some default which could conflict with a users existing network. This is simply omitted in future as its no longer applicable